libloki/proof-of-work.js

```diff
@@ -2,9 +2,17 @@ const hash = require('js-sha512');
 const bb = require('bytebuffer');
 const { BigInteger } = require('jsbn');
 
+module.exports = {
+  calcTarget,
+  incrementNonce,
+  bufferToBase64,
+  bigIntToUint8Array,
+  greaterThan,
+};
+
 const NONCE_LEN = 8;
 // Modify this value for difficulty scaling
-let NONCE_TRIALS = 1000;
+let NONCE_TRIALS = 10;
```

Comment:
I would prefer having both production and development constants defined up here:
```js
let NONCE_TRIALS = 1000;
let DEV_NONCE_TRIALS = 10;
```
and then below
```js
let nonce_trial = development ? DEV_NONCE_TRIAL : NONCE_TRIAL;
```

How does that sound?

Reply:
should be `const` btw :p

Reply:
Hm can you think of how to do that with testing each function individually? Cause that's how it was done before but that check happened what you send an ipc message
I couldn't think of a clean way to do it so just set the default to dev level but I agree it feels gross